lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1826,6 +1864,93 @@ def cmd_db_rebuild_cache_help
     print_line
   end
 
+  def cmd_db_save_help
+    print_line "Usage: db_save <options> <name>"
```

Comment:
Minor: consider changing "<options>" to "[options]" to indicate that it is optional rather than required

Reply:
Thank you for breaking this anti-pattern in Framework! The convention is good. :)